docs/rules/exit-or-cycle-in-unlabelled-loop.md

```diff
@@ -0,0 +1,13 @@
+# exit-or-cycle-in-unlabelled-loop (C142)
+This rule is unstable and in [preview](../preview.md). The `--preview` flag is required for use.
+
+## What does it do?
+Checks for `exit` or `cycle` in unnamed `do` loops
+
+## Why is this bad?
+Using loop labels with `exit` and `cycle` statements prevents bugs when exiting the
+wrong loop, and helps readability in deeply nested or long loops. The danger is
+particularly enhanced when code is refactored to add further loops.
+
+## Settings
+See [nested-loops-only](../settings.md#check_exit-unlabelled-loops_allow-unnested-loops)
```

Comment:
```suggestion
See [allow-unnested-loops](../settings.md#check_exit-unlabelled-loops_allow-unnested-loops)
```

Reply:
Oops! :facepalm: Checked the link actually worked, didn't check this!